src/index.ts

```diff
@@ -466,8 +465,7 @@ export async function getGraphQLParams(
   request: Request,
 ): Promise<GraphQLParams> {
   // `url` is always defined for IncomingMessage coming from http.Server
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const urlData = new URLSearchParams(request.url!.split('?')[1]);
+  const urlData = new URLSearchParams(request.url?.split('?')[1]);
```

Comment:
`request.url` should always be defined in the context we're using it, so I'm not sure how the coverage isn't dropping with the use of conditional chaining here :/

Reply:
@danielrearden Thanks, corrected `Request` type to reflect that.